packages/kerosene/package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "main": "lib/index.js",
   "module": "es/index.js",
+  "sideEffects": false,
```

Comment:
how do we ensure this is always the case? what would it take for one of us to accidentally make some side effect happen?

Reply:
[Webpack's docs on `sideEffects`](https://github.com/webpack/webpack/tree/master/examples/side-effects) do a pretty good job of explaining this, I think.
As an example, putting a top-level `console.log` in a module would violate this, but would probably not cause issues. On the other hand, modifying something in the global state at the top level (like adding an event listener, for example), might cause inconsistent behaviour.
If we stick to mostly functional principles, or at least keep global state modification inside functions, we should be fine.